verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -46,6 +46,26 @@ Signature CreateGlobalSignature(absl::string_view file_path) {
   return Signature(file_path);
 }
 
+// From the given list of anchors returns the list of Anchor values.
+std::vector<std::string> GetListOfReferencesfromListOfAnchor(
+    const std::vector<Anchor>& anchors) {
+  std::vector<std::string> references;
+  for (const auto& anchor : anchors) {
+    references.push_back(anchor.Value());
+  }
+  return references;
+}
+
+// Returns the list of references from the given anchors list and appends the
+// second Anchor to the end of the list.
+std::vector<std::string> GetListOfReferences(const std::vector<Anchor>& anchors,
```

Comment:
I can't tell without caller context, but is the string-copying necessary?  if so, explain here.

Long-term goal: eliminate nearly all string copying with memory lifetime guarantees.

Reply:
The main goal here is to concatenate the references and get them as a one vector. 
what do you want to improve here or comment about?